.travis.yml

```diff
@@ -80,12 +82,24 @@ jobs:
 
 
     # Build statically linked release binaries with gcc 6.3 on Alpine Linux
-    # (inside chroot). If building a tagged commit, then deploy release tarball
+    # (inside chroot). If building a release tag, then deploy release tarball
     # to GitHub Releases.
     - &build-alpine
       <<: *test-alpine
       stage: build
       env: ARCH=x86_64
+      before_install:
+        # XXX: This is ugly hack to skip this job (and all derived) for pull
+        # requests, run it only on master branch and release tags (to cut down
+        # build time). Replace it after Travis finally implement proper support
+        # for conditional jobs or stages.
+        - if [[ "$TRAVIS_PULL_REQUEST" != "false" || ( "$TRAVIS_BRANCH" != "master" && "$TRAVIS_TAG" != 1.* ) ]]; then
```

Comment:
from the travis docs it sounds like TRAVIS_TAG is set when it is a tag. could we check for its existence, as opposed to checking it contains `1.*`? Because there may be tags which are not of that form, like `2.*` some day, but also we have `version_*` tags now as well.

Reply:
What’s the purpose of these `version_*` tags? They looks like redundant for me, so I’ve intentionally excluded them and included only real release tags. Of course I can change it.

Reply:
For clarity purposes, and to avoid similar uncertainties in the future, it might help to have i.e. "e1.37.20" emscripten tags and "v37.0.0" for actual release tags or something like that. At least that'd make it a bit clearer and sticks, at least for release tags, to what's commonly used as tag names.

Reply:
Yeah, maybe the names aren't the clearest. @jirutka: we have two tags in this repo, the `version_*` tags which are binaryen version tags, and the `v1.*.*` which are emscripten version tags (their goal is to make it easy to integrate with emscripten, but they aren't enough by themselves since binaryen can be used without emscripten too).

Reply:
Perhaps you could keep the old tags around, but move to a new naming scheme for new tags: `binaryen_*` and `emscripten_*`. Not sure if that makes things better (since new tags have clearer names) or worse (there are now 4 different tag formats). :woman_shrugging:

Reply:
Aha, this is really _very_ confusing… :man_facepalming:
I recommend to use standard versioning scheme for binaryen releases following [SemVer](http://semver.org/) (binaryen provides C API, so it should also clearly communicate its stability and reflect (non)breaking changes in version number). This is typically prefixed with “v” for tags. And then create tags like `emscripten-x.y.z`, if you really need them.

Reply:
Both proposals sound reasonable to me.

Meanwhile, can we not block this PR on that, and make it not care about the specific name of tags, just whether something is a tag or not?

Reply:
I’ve just updated the PR.